### DISCUSSION TITLE
مرض سیاسی

### DISCUSSION THREAD

**Discussion by BeanVortex** [WeAreMahsaAmini/Core#312] [Ideas]
گیتهاب قراره یه محیط حرفه ای باشه نه یه محیط برای ارضای غرایض سیاسی تون. هشتگ های توییتر و استوری های اینستا بس نبود که اومدید اینجارم بگند بکشید؟
فردا اینجارم ببندن و از کارا عقب بمونیم، مقصر شمائید یا حکومت؟ 
بی وجود نباشید

**Comment by AnonymousMaximus2** [comment]
اگه خارج از ایران هستی که ببندن مهم نیست نگران دیگران نباش

اگه داخل ایران هستی با فیلتر شکن وارد شو

اگر هم برای حکومت کار میکنی برات دعا میکنم خدا شفات بده


اگر هم نمیدونی جریان چیه خدمتت عرض کنم که این یک پروژه سایت اینترنتی بود که کم کم توی این چند روز داره رشد میکنه و از توش دهها ایده و پروژه در اومده و هزار نفر لایک کردن و به نظر من پتانسیل زیادی برای جمع شدن برنامه نویسها داره

البته چند نفر هستند که آشنائی با گیتهاب ندارند که خوب همیشه همه جا اینجور افراد هست ولی اکثریت برنامه نویس هستند 

فیلتر شدن برای برنامه نویسها که به راحتی میتوانند فیلتر شکن پیدا کنند یا حتی برای خود سرور وی پی ان راه اندازی کنند فقط یک جوک هست.

راستی اون هواپیمایی که عکسش را توی پروفایلت گذاشتی جنگنده ایرانی است یا مال کدام کشور است؟
